pkg/host/internal/sriov/sriov.go

```diff
@@ -1029,6 +1047,40 @@ func (s *sriov) setEswitchModeAndNumVFs(pciAddr string, desiredEswitchMode strin
 	return nil
 }
 
+// setEswitchModeAndNumVFsIce configures PF eSwitch and sriov_numvfs in the following order:
+// a. set eSwitchMode to `switchdev` if requested
```

Comment:
we need to reset vf's to 0 before setting switchdev mode, is this handled?

Reply:
I believe so, otherwise driver will throw an error I believe. 
I can double check this tomorrow to confirm

Reply:
I can see in some docs that we must have 0 VF's before switching to switchdev mode.